src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java

```diff
@@ -194,4 +194,8 @@ public void verifyMessageAboutNewPassword(String userName) {
 			true
 		);
 	}
+
+	public void verifyFBButtonVisible() {
+		Assertion.assertTrue(checkIfElementOnPage(formConnectWithFbButton));
```

Comment:
When rely on just the base class version of ```verifyFBButtonVisible()``` though, we'll have to make sure that all subclasses use the same name for the instance variable it checks. Here we should rename ```formConnectWithFbButton``` to ```facebookSignUpButton``` on this line, as well as [here](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java#L42) and [here](https://github.com/Wikia/selenium-tests/blob/master/src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java#L131).